docs/content/reference/migration/migration-0-15.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Migrating from 0.14 to 0.15
+order: 150
+---
+
+## `InstanceKey` removed from our logging APIs
+In PR [#5395](https://github.com/rerun-io/rerun/pull/5395) we removed the `InstanceKey` component from all our archetypes.
+
+What were instance keys?
+
+In Rerun, each entity can be a batch of _instances_. For instance, a point cloud can be one entity, and each point is an instance of that entity.
```

Comment:
The use of "can be" is kinda weird -- suggesting that this anything other than a fact
```suggestion
In Rerun, each entity can be a batch of _instances_. For instance, a point cloud is a single entity, while each point is an instance of that entity.

Reply:
You can log every point as its own entity and call it a point cloud